day6/index.js

```diff
@@ -0,0 +1,45 @@
+$(() => {
+  const dict = ['apprehensive', 'barricade', 'condition', 'dictionary', 'fragile', 'grueling', 'habitation', 'illuminate',
+                'jostle', 'kindle', 'luminous', 'materialize', 'outlandish', 'plenteous', 'question', 'restaurant',
+                'seperate', 'turtle','umbrella','versatile','weather','xray','yoyo','zepalz']
+
+  const randomWord = () => {
+    let random = dict[Math.floor((Math.random() * (dict.length)))]
+    for (var i = 7; i > 0; i--) {
```

Comment:
Do not use `var` because `var` is not a `block scope`, meaning when you are finished with the `for` loop, you can still access variable `i` anywhere in the function `randomWord` even it's not in the loop. In `for` loop, it is better to use `let`.